database/config.js

```diff
@@ -0,0 +1,4 @@
+const knex = require("knex")
+const knexfile = require("../knexfile")
+
+module.exports = knex(knexfile.development)
```

Comment:
In a future PR this will need to change to be dynamic based on the `NODE_ENV` or `DB_ENV`